import/Make.defs

```diff
@@ -77,7 +77,14 @@ endif
 # Link with user libraries
 
 ifeq ($(CONFIG_BUILD_KERNEL),y)
-  LDLIBS = -lapps -lxx -lmm -lc -lm -lproxies
+  LDLIBS = -lapps -lxx -lmm -lc -lproxies
+ifneq ($(CONFIG_LIBM),y)
+  LIBM = ${shell "$(CC)" $(ARCHCPUFLAGS) -print-file-name=libm.a}
```

Comment:
```suggestion
  LIBM = ${shell "$(CC)" $(ARCHCPUFLAGS) --print-file-name=libm.a}
```
Here and below. This is just to keep the same style as in main OS repo

Reply:
Done